telemetry/jetbrains/src/test/resources/testGeneratorInput.json

```diff
@@ -47,6 +47,11 @@
       "description": "called when invoking lambdas remotely",
       "unit": "None",
       "metadata": [{ "type": "lambdaRuntime", "required": false }, { "type": "inttype" }]
+    },
+    {
+      "name": "no_metadata",
```

Comment:
why not give it a prefix/"namespace"? e.g. `lambda_no_metadata` ?

Reply:
this is just test input, this puts it in the namespace "no" though